### PR TITLE
Presentation Draft

### PR DESCRIPTION
**Pull request by samgdotson**
Adds a draft of the presentation to the repository.

### CODE REVIEW

**Comment by robfairh** [comment]
@samgdotson I have several comments:
* The presentation looks really nice.
* I haven't heard your talk yet, but the first of your conclusions confuses me. You say that replacing ABBOTT with nuclear would resolve the University carbon goals, but the figures presented in the results do not consider such a scenario. The figure in slide 17 ('temoa_uiuc_nuclear.png') shows ABBOT as an input in your model. And then, the second figure in slide 22 ('scenario3_generation_stm.png') shows ABBOT as a source of steam.
* I think there are a few images that you forgot to add and commit (for example 'temoa_uiuc_nuclear.png').
* I would delete pres/images/ it has a bunch of figures that you are not using.
* This PR is huge, as there are a few files that are part of #39. I think before merging this PR I should review that one. That will make this PR easier to review.

**Comment by samgdotson** [comment]
1) The grid model in slide 17 is an overview -- says nothing about what technology is actually being used to produce steam/electricity.
2) Scenario 3 limits the capacity of the nuclear plant to 100 MWth, not enough to fully replace Abbott Power Plant. Thus in order to meet demand for steam and electricity, the model must use Abbott. In scenarios 1 + 2, 100% of steam is produced by nuclear because those are the unconstrained scenarios. 
3) I will delete that folder, they're from the presentation template. 
4) The PR is quite large, but most of the files are figures.

**Comment by robfairh** [comment]
I merged #39 and that caused a few conflicts. Fix those and I will merge this PR.

**Review by robfairh — APPROVED** [review]

**Pull request merged**
